src/test/java/com/ordestiny/tdd/kata/RPNCalculatorTest.java

```diff
@@ -0,0 +1,33 @@
+package com.ordestiny.tdd.kata;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+
+class RPNCalculatorTest {
+
+    private static RPNCalculator rpnCalculator;
+
+    @BeforeAll
+    public static void init() {
+        rpnCalculator = new RPNCalculator();
+    }
+
+    @ParameterizedTest
+    @MethodSource("testData")
+    void calculate(String postExpr, int expected) {
```

Comment:
Hi , 
it seems you didn't pass the coverage check.
[ant:jacocoReport] Rule violated for class com.ordestiny.tdd.kata.RPNCalculator: lines covered ratio is 0.85, but expected minimum is 0.95

> Task :jacocoTestCoverageVerification FAILED
FAILURE: Build failed with an exception.
2 actionable tasks: 1 executed, 1 up-to-date

* What went wrong:
Execution failed for task ':jacocoTestCoverageVerification'.
> Rule violated for class com.ordestiny.tdd.kata.RPNCalculator: lines covered ratio is 0.85, but expected minimum is 0.95

you could use gradle tasks
such as 
gradle task jacocoTestReport to see, which lines are not covered

Reply:
@fengyuanyang addressed